librz/include/rz_core.h

```diff
@@ -434,6 +434,15 @@ typedef struct rz_bin_sym_names_t {
 	char *methflag; ///< methods flag sym.[class].[method]
 } RzBinSymNames;
 
+/**
+ * \brief Message for `pdJ`
+ */
+typedef struct rz_analysis_disasm_text_t {
+	ut64 offset;
+	ut64 arrow;
+	char *text;
+} RzAnalysisDisasmText;
```

Comment:
Is this really just for `pdJ` command? If so, is it really necessary?

Reply:
Yes